### ISSUE TITLE
Update react-scripts for create-react-app

### ISSUE THREAD

**Issue by mvbattan** [sbalay/react-project-template#5]
Because `jsx-a11y` linter validations are failing.

**Comment by sbalay** [comment]
8f95f3cf8188b7252535a672347f53813a2e3ad9
